source/_posts/2020-11-30-community-highlights.markdown

```diff
@@ -0,0 +1,127 @@
+[p3]: https://www.troyhunt.com/iot-unravelled-part-3-security/
+[p4]: https://www.troyhunt.com/iot-unravelled-part-4-making-it-all-work-for-humans/
+[p5]: https://www.troyhunt.com/iot-unravelled-part-5-practical-use-case-videos/
+
+## Framing a wall-mounted tablet
+
+If you are a bit like me, you probably want to wall mount a tablet someday
+(or you already have it of course 🥴). Doing that in a nice, elegant and pretty
+way is often the hard part. Especially if you only have brick walls (like me),
+which makes it hard to hide cable or recess the tablet.
+
+Next I came across 2 posts on on the [Home Assistant subreddit][reddit], which,
+if you haven't done already, should totally join.
+
+This one popped up, made by ialex87, which uses a
+[3D printed frame from Thingiverse][stl] to which he glued a wooden frame
+to make it look like a photo frame.
+
+<blockquote class="reddit-card" data-card-created="1606506563">
+<a href="https://www.reddit.com/r/homeassistant/comments/joncw6/nothing_much_but_my_honest_work/">Nothing much, but my honest work!</a> from <a href="http://www.reddit.com/r/homeassistant">r/homeassistant</a>
+</blockquote>
+
+Next, Invinciberry shared his version:
+
+<blockquote class="reddit-card" data-card-created="1606506801">
+<a href="https://www.reddit.com/r/homeassistant/comments/jw6g9m/now_lets_wall_mount_this_thing/">Now let's wall mount this thing.</a> from <a href="http://www.reddit.com/r/homeassistant">r/homeassistant</a>
+</blockquote>
+<script async src="//embed.redditmedia.com/widgets/platform.js" charset="UTF-8"></script>
+
+Both look very slick! Thanks for sharing!
+
+[stl]: https://www.thingiverse.com/thing:2853298
+
+## Arm your alarm by brushing your teeth...
+
+Automatically arm your alarm, when Home Assistant detects you are
+using your toothbrush... Why? Because we can! 😎
+
+<blockquote class="twitter-tweet">
+<p lang="en" dir="ltr">Have you ever seen a house alarm system controlled by a toothbrush? Here our <a href="https://twitter.com/VanderbiltInd?ref_src=twsrc%5Etfw">@VanderbiltInd</a> alarm is integrated into <a href="https://twitter.com/home_assistant?ref_src=twsrc%5Etfw">@home_assistant</a> . Our toothbrush is ble enabled, it’s state is read by an ESP32 chip and sent into <a href="https://twitter.com/home_assistant?ref_src=twsrc%5Etfw">@home_assistant</a> <a href="https://t.co/T9XGi1Zjmb">pic.twitter.com/T9XGi1Zjmb</a></p>&mdash; Ian (@imduffy15) <a href="https://twitter.com/imduffy15/status/1329569523855478784?ref_src=twsrc%5Etfw">November 19, 2020</a>
+</blockquote>
+<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+## Home Assistant Conference
+
+This isn't a highlight yet, but without a doubt, it is definitely is going to
+become one of the highlights of this year: The Home Assistant Conference.
+
+<a href="/conference"><img src='/images/conference/conference-header.png' alt='Home Assistant Conference header' class='no-shadow'></a>
```

Comment:
Can we embed a screenshot of the schedule instead

Reply:
Yes, we can now 👍  (the screenshot was published after this PR).

Will adjust.